### ISSUE TITLE
Comparison with Bootswatch Paper

### ISSUE THREAD

**Issue by dandv** [Daemonite/material#49]
Since this is built with Bootstrap, it might be worth mentioning in README the extra features it offers vs. [Bootswatch Paper](https://bootswatch.com/paper/)  (see #16).

**Comment by modius** [comment]
I'm not sure what the benefits to this project would be.  We don't have time to compare and contrast every Material design project with our own. Our first efforts will be to better document various aspects of our own project prior to promoting others.

Thank you for the suggestion.

**Comment by dandv** [comment]
Maybe this is just me, but if I were to start, say, a UI components library, I would first look at some of the most popular [existing ones](http://stackoverflow.com/questions/200284/what-are-alternatives-to-extjs), and come up with solid reasons why I'd invest the effort to make yet another widgets library. A few comparisons would certainly be faster than building a library from scratch, and I'd learn how to position the product to attract the right target audience. For example, in Material's case, mentioning Bootstrap in the README is a big plus. Not mentioning the advantages vs. Bootswatch Paper risks losing users who'll just say "Oh it's just another Material theme for Bootstrap, skip".

**Comment by modius** [comment]
We started our project about the time Google released the Material Spec mid last year, and released our first project in November 2014. We decided to release the UI as an open source framework in February 2015.

Beyond a few snippets, there wasn't anything in the way of an HTML library for Material at the time.

Our target audience has really been ourselves.  Though given the accidental interest in our project, we're trying to find the time to make it more approachable for others.
